src/ray/rpc/grpc_server.h

```diff
@@ -118,6 +118,8 @@ class GrpcServer {
   void RegisterService(GrpcService &service);
   void RegisterService(grpc::Service &service);
 
+  grpc::Server &GetServer() { return *server_; }
```

Comment:
Mark it as // test-only?

Reply:
Although only used in testing so far, but I think it can be a public API in the future.
For example, raylet can set worker-pool as SERVING/NOT_SERVING.